src/main/java/net/glowstone/util/LibraryManager.java

```diff
@@ -52,13 +88,20 @@ public void run() {
         }
 
         downloaderService.execute(new LibraryDownloader(
-                "org.xerial", "sqlite-jdbc", "3.21.0", ""));
+                "org.xerial", "sqlite-jdbc", "3.21.0",
+                "347e4d1d3e1dff66d389354af8f0021e62344584", DEFAULT_HASH_ALGORITHM));
```

Comment:
I think using DEFAULT_HASH_ALGORITHM here actually makes the code a bit less clear. With that, I have to look up that the default hash algorithm is SHA1, and if we ever need to change that then all these hashes will have to change. If you  used HashAlgorithm.SHA1, it is immediately clear what type of hash this is and allows us to switch between hashes without changing all the entries at once.